spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitTemplateCustomizer.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2012-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.amqp;
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+
+/**
+ * Callback interface that can be used to customize a {@link RabbitTemplate}.
+ *
+ * @author dang zhicairang
+ * @since 2.3.0
+ */
+@FunctionalInterface
+public interface RabbitTemplateCustomizer {
+
+	/**
+	 * Customize the {@link RabbitTemplate}.
```

Comment:
Minor note:
  It would be nice to have a bit more info of the method. What excactly it does?

Reply:
I will add more info for it.